src/main/java/com/cx/plugin/cli/configascode/SastConfig.java

```diff
@@ -22,9 +22,10 @@ public class SastConfig {
     @Optional
     private int high;
     @Optional
-    private boolean isOverrideProjectSetting;
-
-
+    private boolean isOverrideProjectSetting;    
+    @Optional
```

Comment:
Why is this field to be added in SastConfig.java which is in configascode package.
This feature we are not supporting from config as code perspective.